docs/code_guidelines.md

```diff
@@ -294,3 +294,19 @@ cy.getBySel("reservation-success-title-text")
   // Another assertion.
   .and("contain", "Material is available and reserved for you!");
 ```
+
+## Writing Unit Tests
+
+We are using Vitest as framework for running unit tests.
+By using that we can test functions (and therefore also hooks) and classes.
+
+### Where do I place my tests?
+
+They have to be placed in src/tests/unit.
```

Comment:
```suggestion
They have to be placed in `src/tests/unit`.
```

Reply:
I wonder if we could place test files along with the code that they test like we do for Cypress tests.

I guess there is a possibility for mixing up the two frameworks if all test files use the format `[scope].test.ts`. If we want to separate the two then one approach might be to use `vite` for `src/core` and Cypress for `src/apps` and `src/components`. That would also clearly communicate the responsibility of the different tools and folders. It might be too strict though.